### DISCUSSION TITLE
Reverse course

### DISCUSSION THREAD

**Discussion by guerrillian** [Courseplay/Courseplay_FS22#2697] [Ideas]
Hi!

What about an option that enables the reverse route? This way we could use a tractor on another ongoing work, and finish earlier!

Thanks.

**Comment by Tensuko** [comment]
No Idea what you want to do...
What do you mean with reverse route?
You mean start the course from the other side?
This was requested already some times and we have this somewhat on our list.
If you want to finish your work faster, use 2 or more on the same field with multi tool course!

**Reply by guerrillian** [reply]
reverse route to allow 2 tractors to run against one another.
I use multi tool but sometimes I need to wait for one tractor to finish some work to then start another with multitool. With a reverse route it would be possible to finish the 2nd work starting the 2nd tractor by the end when the other tractor already started by the start

**Reply by Tensuko** [reply]
As said, is in planning but not sure when and how we fet it to work.
